### DISCUSSION TITLE
Best way to close a tooltip

### DISCUSSION THREAD

**Discussion by loxy** [taiga-family/taiga-ui#706] [Q&A] [answered]
Hi again!

What is the best way to close a tooltip when a click on the same elements occurs? I implement a side menu. When it is collapsed, there are only icons. Every icon has a tooltip. When I click on the element a dropdown menu appears. But at the same moment the tooltip should disappear. But I didn't found a way to close the tooltip. Any hints?

Best regards,
Kersten

**Comment by waterplea** [comment, marked as answer]
Best way would be to make hint computable:
```
get hint(): string {
  return this.isMenuOpen ? '' : 'My actual hint';
}
```

**Reply by loxy** [reply]
Ok. Thank you. But after thinking about it, I realized that this is not good UX anyway. So I talked to our designer and we will remove this behavior. It definitely makes no sense.
